.github/workflows/typescript.yml

```
@@ -1,56 +1,103 @@
+        with:
+          bun-version: ${{ env.BUN_VERSION }}
+      - name: Install dependencies
+        run: bun install
+      - name: Run tests with coverage
+        run: bun run test:ci
+      - name: Upload test results and coverage
+        uses: actions/upload-artifact@v4
+        with:
+          name: test-results
+          path: |
+            test-results.xml  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)